src/test/java/io/vertx/rxjava/ext/mongo/MongoConcatWithTest.java

```diff
@@ -0,0 +1,128 @@
+package io.vertx.rxjava.ext.mongo;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.mongo.BulkOperation;
+import io.vertx.ext.mongo.FindOptions;
+import io.vertx.ext.mongo.MongoClientBulkWriteResult;
+import io.vertx.ext.mongo.MongoTestBase;
+import io.vertx.rxjava.core.Vertx;
+import rx.Observable;
+import rx.Single;
+import rx.exceptions.Exceptions;
+
+public class MongoConcatWithTest extends MongoTestBase {
+
+  private MongoClient mongoClient;
+
+  private int messageCount = 55;
+  private int bulkWriteSize = 10;
+
+  private OffsetDateTime now = OffsetDateTime.of( 2018, 1, 4, 10, 25, 0, 0, ZoneOffset.UTC );
```

Comment:
If this OffsetDateTime is truly a constant then make it one, perhaps change variable name as well as it's not really 'now'